features/content-visibility.yml.dist

```diff
@@ -39,3 +42,12 @@ compat_features:
   - api.ContentVisibilityAutoStateChangeEvent
   - api.ContentVisibilityAutoStateChangeEvent.ContentVisibilityAutoStateChangeEvent
   - api.ContentVisibilityAutoStateChangeEvent.skipped
+
+  # baseline: false
+  # support:
+  #   chrome: "108"
+  #   chrome_android: "108"
+  #   edge: "108"
+  #   firefox: "130"
+  #   firefox_android: "130"
+  - api.Element.contentvisibilityautostatechange_event
```

Comment:
Can you confirm this is actually not supported in Safari? I'm wondering if this just got missed in BCD. We could potentially avoid a compute_from if it is actually supported.

Reply:
Per our testing, this is supported, so I opened https://github.com/mdn/browser-compat-data/pull/25186. 

This will mean we won't need a compute_from, but we'll be blocked on that issue. @ddbeck  Can you help with the BCD issue when you have a chance?

Reply:
This is merged on BCD, and should be included in BCD 5.6.19.